packages/documentation-framework/components/example/example.js

```diff
@@ -77,8 +78,6 @@ export const Example = ({
   liveContext,
   // Content that appears between h3 and code block to explain example
   children,
-  // Show dark theme switcher on full page examples
-  hasDarkThemeSwitcher = process.env.hasDarkThemeSwitcher
```

Comment:
Why did you remove this? I think we want the dark theme switcher on full pages if the environment sets `hasDarkThemeSwitcher` to true.

And it is also seeming like the full page demos are not working in the review? 
https://patternfly-org-pr-3126-v4.surge.sh/v4/components/notification-drawer/react-demos/basic/
https://patternfly-org-pr-3126-v4.surge.sh/v4/components/page/react-demos/sticky-section-group-using-pageheader/

Reply:
Ah, not intentional. Will add this back.

Reply:
Both of those links point to their respective pages now fine it appears.